### DISCUSSION TITLE
Welcome to google-searchconsole Discussions!

### DISCUSSION THREAD

**Discussion by joshcarty** [joshcarty/google-searchconsole#27] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tomfbush** [comment]
Just realised I should have maybe posted this possibly non "issue" in here! https://github.com/joshcarty/google-searchconsole/issues/28

I'm Tom and I work in SEO and analytics in the UK :) thanks for this!

**Reply by joshcarty** [reply]
Thanks for this Tom! Great to meet you and I'll reply in the issue itself.
